src/utils/collapseSourcemaps.ts

```diff
@@ -145,20 +146,13 @@ class Link {
 	}
 }
 
-export default function collapseSourcemaps(
-	bundle: Chunk,
-	file: string,
-	map: DecodedSourceMap,
-	modules: Module[],
-	bundleSourcemapChain: DecodedSourceMapOrMissing[],
-	excludeContent: boolean
-) {
-	function linkMap(source: Source | Link, map: DecodedSourceMapOrMissing) {
+function linkMapper(graph: Graph) {
```

Comment:
I would prefer something like `getLinkMap` so that the function name starts with a `verb`.